mender/install_mender.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -e
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run with sudo"
+  exit
+fi
+
+
+
+function get_mender()
+{
+   [ ! -d ./mender-connect ] && git clone -b 2.0.1 https://github.com/mendersoftware/mender-connect.git ./mender-connect
```

Comment:
```suggestion
   [ ! -d ./mender-connect ] && git clone -b 2.0.1 https://github.com/mendersoftware/mender-connect.git ./mender-connect
   true
```

This function sends a false command whenever the mender-connect is already cloned. It makes the script to stop execution before running the other commands.